theme/type/responsiveType.less

```diff
@@ -9,13 +9,23 @@
   // Concatenate type hierarchy variable name
   @type-scale: "@{variant}-type-scale";
   @row-span: "@{variant}-type-row-span";
+  @weight: "@{variant}-type-weight";
   @mobile-type-scale: "@{type-scale}-mobile";
   @mobile-row-span: "@{row-span}-mobile";
+  @mobile-weight: "@{variant}-type-weight-mobile";
 
   @media only screen and (max-width: @max-breakpoint) {
     .__baselineType(@@mobile-type-scale, @@mobile-row-span);
+
+    & when (not(@@mobile-weight = 400)) {
```

Comment:
Can you unpack the logic on this one? 400 is normal, but is there a reason not to double apply it?

(also good work to CSS/Less on that `=` syntax, why work the way a dev would expect you to?)